napari/utils/_tests/test_interactions.py

```diff
@@ -6,10 +6,9 @@
 @pytest.mark.parametrize(
     'shortcut,reason',
     [
-        ('Ctrl-A', 'Ctrl instead of Control'),
-        ('Ctrl+A', '+ instead of -'),
-        ('Ctrl-AA', 'AA make no sens'),
-        ('BB', 'BB make no sens'),
+        ('Atl-A', 'Alt misspelled'),
+        ('Ctrl-AA', 'AA makes no sense'),
```

Comment:
Just want to double-check that the removed `Ctrl+A` test case was intentional.

Reply:
yes, we’re able to parse both `-` and `+` as separators now